Project_2/djangoProject/cars/views.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib import messages
 from django.contrib.messages.views import SuccessMessageMixin
-from django.contrib.auth.decorators import login_required
+from django.contrib.auth.decorators import login_required, permission_required
```

Comment:
`permission_required` does not seem to be used anywhere else. If it's not needed, please remove it.